.travis.yml

```diff
@@ -6,20 +9,26 @@ cache:
 matrix:
   include:
     - node_js: '7'
-      script: npm run pretest
-      env: CI=pretest
+      script: npm run preci
+      env: CI=preci
```

Comment:
By making the above change of removing the `pretest` Travis CI job in favour of your `preci` job you've removed the only job that ran ESLint as part of any Travis CI job.